docs/_blog/road_to_bdk_1.md

```diff
@@ -0,0 +1,98 @@
+You need to change the `Blockchain` trait to fetch that extra bit of information. And the list goes on and on.
+
+Since making changes to these traits is always so painful for us and our downstream users, this ended up delaying or considerably slowing down the development of new features in BDK.
+
+bdk\_core tries to minimize the (ab)use of traits, and this will help immensely when trying to provide a stable API for our users.
+
+### Upstreaming our code
+
+BDK internally implements many features that could be useful to other projects as well. While working on this integration we will also try to upstream some of our code to the relevant crates, mainly [`rust-miniscript`].
+
+This has essentially three benefits:
+
+1. A new set of eyes will take a look at the code, potentially spotting issues or suggesting improvements
+2. This will lower the amount of code that we have to maintain ourselves
+3. Other people can benefit from our code, which was previously tightly integrated into BDK and hard to re-use
+
+### Partially Syncing a Wallet
+
+This single point was the main reason the bdk\_core project was kickstarted, and it means giving our users the ability to incrementally sync a wallet over time instead of working in single big batches.
+
+This is explained very well in the [first bdk\_core post], so I won't go into details here. Think of this as a much more flexible way to sync a wallet, which in turn will allow us to simplify our current implementation
+of blockchain backends like compact block filters, and also offer a better API for our users.
+
+### `no_std`
+
+bdk\_core is built with `no_std` in mind since the beginning, something we've been wanting to support in BDK [for a long time](https://github.com/bitcoindevkit/bdk/issues/205). Being more modular means the "core" module doesn't really need that many dependencies,
+and this really simplifies the `no_std` work.
+
+This will allow the main components of BDK to work on embedded hardware as well, making it possible to use the library as a foundation for any Bitcoin hardware device like hardware wallets.
+
+### Lower MSRV
+
+Removing many of our current dependencies from the core components of BDK will also allow us to lower our MSRV considerably, which in turn will allow BDK to compile on older distros or entirely different toolchains like
+[`mrustc`], which usually don't keep up with `rustc` in terms of language features.
+
+## Architecture
+
+Roughly speaking, after integrating bdk\_core into BDK the architecture will look like this:
+
+- bdk\_core: this crate will contain all the low level components of a Bitcoin wallet. For example, using this low level API it will be possible to keep track of arbitrary scripts (without the limitations
+  of descriptors) or apply individual blocks to the state of the wallet
```

Comment:
> without the limitation of descriptors

I think this affirmation needs at least a footnote  explaining which are these limitations

Reply:
Not only that, also scripts that can't be expressed as descriptors